### ISSUE TITLE
Stuck at load... again

### ISSUE THREAD

**Issue by JasonMcRay** [jakimfett/Minechem#712]
The issue I was having before (#637 and #511) is again back. I am still using the same version as in #637 which should have fixed it, but it did not :(

**Comment by jakimfett** [comment]
Do you still get this with the latest builds of Minechem?

**Comment by JasonMcRay** [comment]
Yes i am still having the same issue with latest build. I recently bought new PC and its still happening on it, so i doubt the issue is my PC.

**Comment by jakimfett** [comment]
Could you provide a download of your pack, please? I know you did before, I assume its had some updates since then, though...

**Comment by JasonMcRay** [comment]
http://ftb.cursecdn.com/FTB2/privatepacks/InfiTech2/2_2_1_1/InfiTech2.zip

It has version 397, but still when I update to 399 the stuck is happening.

2015-06-02 17:18 GMT+02:00 jakimfett <notifications@github.com>:
